common/content/options.js

```diff
@@ -41,7 +41,7 @@ const Option = Class("Option", {
 
         // add no{option} variant of boolean {option} to this.names
         if (this.type == "boolean")
-            this.names = array([name, "no" + name] for (name in values(names))).flatten().__proto__;
+            this.names = util.Array.flatten(Array([name, "no" + name] for (name in values(names)))).__proto__;
```

Comment:
Ah, I see that function being defined in `base`. Going to close this PR. We'll get around to it at some point unless you want to give it a shot :).